rules-reviewed/quarkus1/quarkus1-10/quarkus1-11-logging.windup.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="quarkus1-11-logging"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            Quarkus project Release 1.11 changes to logging properties configuration
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="quarkus1" versionRange="(,10]"/>
+        <targetTechnology id="quarkus1" versionRange="[11,)" />
+    </metadata>
+    <rules>
+        <rule id="quarkus1-11-logging-00000">
+            <when>
+                <filecontent pattern="{logLevel}" filename="application.properties"/>
+            </when>
+            <perform>
+                <hint title="Log `min-level`" effort="1" category-id="potential" >
+                    <message>If you are using `DEBUG` or `TRACE` log levels, we made an important change in our logging layer: 
+                        a new build-time `min-level` configuration property was introduced that sets the minimum log level you will be able to use at runtime.
+                        So if you are logging at `DEBUG` level for some loggers, setting `min-level` to `DEBUG` (or `TRACE`) is required.</message>
+                    <link title="Quarkus - Configuring Logging" href="https://quarkus.io/guides/logging" />
+                    <link title="Quarkus - Migraton Guide 1.11" href="https://github.com/quarkusio/quarkus/wiki/Migration-Guide-1.11" />
+                </hint>
+            </perform>
+            <where param="logLevel">
+                <matches pattern="(.level=DEBUG|.level=TRACE)"/>
```

Comment:
```suggestion
                <matches pattern="(.level\s*=\s*(DEBUG|TRACE))"/>
```
This way you can cover also the variants having spaces before or after the "="  . You can see it as 1st example in the properties guide : https://quarkus.io/guides/config

Reply:
I'm even thinking if we should also consider level "ALL" as it is one of the levels accepted : https://quarkus.io/guides/logging